scripts/deploy_launcher.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z "$1" ]]; then
+  echo "RUNNER_URL is mandatory"
+  exit 1
+fi
+
+RUNNER_URL=$1
+IMAGE_TAG="${2:-v1.1.0}"
```

Comment:
Will this become a pain to manage? Everytime we do a new release we have to update the version number?

Reply:
It's only a default so doesn't matter to much since you can always set your own tag?

Reply:
yeh i get that, i just wonder how when we will every update the default in this script. It's probably more of an issue for the runner script as there are more frequent releases to that. We will just get lazy and always have an old version deployed locally

Reply:
maybe if the master build also pushed latest then we could default to that?

Reply:
That was the alternative, but then got the feeling that we didn't want to push with the tag `latest` but I guess we can push that as well then default it here.

Reply:
yeh we could push with a tag of latest and the commit ref?

Reply:
Done